DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h

```diff
@@ -159,6 +161,11 @@ class CTPPSPixelLocalTrack
     inline void setValid(bool valid) { valid_ = valid; }
 
     bool operator< (const CTPPSPixelLocalTrack &r);
+
+    inline ReconstructionInfo getRecoInfo() const { return recoInfo_; }
+    inline void setRecoInfo(ReconstructionInfo recoInfo) { recoInfo_ = recoInfo; }
+
+    inline unsigned short getNumberOfPointsUsedForFit() const { return (unsigned short) numberOfPointsUsedForFit_; }
```

Comment:
remind me please why this cast is applied
(or why is the native type not ```unsigned short```).

Reply:
I think this is probably a residual, that I forgot to change (the variable could have been initially defined as int, then moved to unsigned short, since it's more efficient. I'm removing it since it's superfluous